src/models/objectiveTemplate.js

```diff
@@ -14,12 +14,12 @@ export default (sequelize, DataTypes) => {
       ObjectiveTemplate.belongsTo(models.Region, { foreignKey: 'regionId', as: 'region' });
       ObjectiveTemplate.hasMany(models.Objective, { foreignKey: 'objectiveTemplateId', as: 'objectives' });
       ObjectiveTemplate.hasMany(models.GoalTemplateObjectiveTemplate, { foreignKey: 'objectiveTemplateId', as: 'goalTemplateObjectiveTemplates' });
-      ObjectiveTemplate.belongsToMany(models.GoalTemplate, {
-        through: models.GoalTemplateObjectiveTemplate,
-        foreignKey: 'objectiveTemplateId',
-        otherKey: 'goalTemplateId',
-        as: 'goalTemplates',
-      });
+      // ObjectiveTemplate.belongsToMany(models.GoalTemplate, {
+      //   through: models.GoalTemplateObjectiveTemplate,
+      //   foreignKey: 'objectiveTemplateId',
+      //   otherKey: 'goalTemplateId',
+      //   as: 'goalTemplates',
+      // });
```

Comment:
```suggestion
      // ObjectiveTemplate.belongsToMany(models.GoalTemplate, {
      //   through: models.GoalTemplateObjectiveTemplate,
      //   foreignKey: 'objectiveTemplateId',
      //   otherKey: 'goalTemplateId',
      //   as: 'goalTemplates',
      // });
```
```suggestion 
```

Reply:
not sure what I did with the suggestions there, but the idea was deleting the commented out association